### ISSUE TITLE
Lightweight WikiMap Clone

### ISSUE THREAD

**Issue by amaliebarras** [open-austin/project-ideas#69]
### What problem are we trying to solve?

<img width="1071" alt="screen shot 2016-06-20 at 11 14 31 pm" src="https://cloud.githubusercontent.com/assets/11826646/16218021/f18c16d6-373e-11e6-9b3e-79a1584b2b9f.png">

Crowdsourcing maps is awesome and useful for showing demand and interest in things, but they are tough to learn to make, and the [tools](http://wikimapping.com/wikimap/admin/index.php) cost money 

### Who will benefit (directly and indirectly) from this project?

People who want to make maps where users can report things (like b-cycle did) - could be reporting bike lanes they'd like to see, signals to fix, sidewalk needs, etx. 

### Links to any research/data available/articles

Links here.

### What are the next steps (validation, research, coding, design)? 

We should write the Wikimaps people and ask if they care to donate. Otherwise, we can do more research with respect to needs and what's already been done, & map out a workflow & project plan. 

### What help is needed at this time?

Project manager who may want to take this on, designers, backend developers, mappy peeps.

**Comment by luqmaan** [comment]
What are some of the existing tools that do this (or a similar thing)?

- http://wikimapping.com/ - Collaborative, $350
- https://crowdmap.com/map/austindogparks/ - Collaborative, Free
- https://www.google.com/work/mapsearth/products/mymaps.html - Public, Free

**Comment by amaliebarras** [comment]
Sent message to wikiMapping people asking if they'd donate a project to us.

**Comment by amaliebarras** [comment]
Just tried playing around with CrowdMap to make [this](https://crowdmap.com/map/sidewalkpleaseatx) - am finding it impossible to add a location, I think their search is broken?

**Comment by amaliebarras** [comment]
```Hi Amalie,

Thanks for writing. I'm glad you like it. We have a lot of improvements planned.  

To set up a project, you can just register and set up draft projects.  This lets you get 100 comments from 100 people.  So, it's a good way to test and see if it will meet your needs.  Then, I'd be happy to help make the projects active when needed.

Let me know if you have questions.

Steve``` @luqmaan woo new resource fo' us

**Comment by luqmaan** [comment]
I just added a location.

![image](https://cloud.githubusercontent.com/assets/1275831/16277674/955aa0da-3879-11e6-9692-64db57190147.png)

Click `Create Post` and then click the map button.

![image](https://cloud.githubusercontent.com/assets/1275831/16277677/9a08c1d4-3879-11e6-8811-1ba1a0ad005f.png)

![image](https://cloud.githubusercontent.com/assets/1275831/16277654/7b87ea00-3879-11e6-96ed-03d9aea0061a.png)

![image](https://cloud.githubusercontent.com/assets/1275831/16277664/8aaba3be-3879-11e6-8a01-1fc081f5259d.png)

**Comment by amaliebarras** [comment]
@luqmaan Yeah but you just added a random location, I was able to do that too - but I am trying to add a specific location at the intersection of Rainey & Caesar Chavez. I can't search for it and I can't drop a pin where I want it.

**Comment by almereyda** [comment]
I believe uMap could be interesting for you: https://umap.openstreetmap.fr

**Comment by amaliebarras** [comment]
[basically this!](https://github.com/open-austin/project-ideas/issues/80) but what if it worked on desktop too?
